git-link.el

```diff
@@ -269,7 +299,7 @@
 	  dirname
 	  commit))
 
-(defun git-link-gitorious (hostname dirname filename branch commit start end)
+(defun git-link-gitorious (hostname dirname filename _branch commit start _end)
```

Comment:
Oh I see, the warning is emitted due to `lexical-binding: t`.  👍 

Just started trying to use that, actually. I'm a dinosaur on the Emacs front 👴.